### DISCUSSION TITLE
Routes: Shield setup is not add route if $routes is commented out

### DISCUSSION THREAD

**Discussion by alisacorporation** [codeigniter4/shield#1005] [Q&A]
Hi all, I'm using modules, and I have commented out default route in `app/Config/Routes.php` (defined my routes in module) and when I setup shield it is not adding line: 
```php
service('auth')->routes($routes); 
```

It is "bug" or it works as intended? Thanks!

**Comment by kenjis** [comment]
Strictly speaking, it would be a bug.
If anyone can fix it, please send a PR.

**Reply by ifjtoledo** [reply]
my codeigniter installation too
